starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,66 @@
+body{
+	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+	margin: 0;
```

Comment:
Modifying the default CSS rules is a good practice! Some people even took the time to create complete css reseting blocks for you to copy paste in your projects. You can find a nice one here: https://meyerweb.com/eric/tools/css/reset/